docs/experimental-machine-learning/problem-child.md

```diff
@@ -0,0 +1,43 @@
+# ProblemChild in the Elastic Stack 
+
+ProblemChild helps detect anomalous activity in Windows process events by:
+1) Classifying events as malicious vs benign
+2) Identifying anomalous events based on rare parent-child process relationships
+
+An end-to-end blog on how to build the ProblemChild framework from scratch for your environment can be found [here](https://www.elastic.co/blog/problemchild-detecting-living-off-the-land-attacks).
+
+You can also find some supplementary material for the blog and examples [here](https://github.com/elastic/examples/tree/master/Machine%20Learning/ProblemChild)
+
+We also released a blog about getting started with ProblemChild using the CLI and Kibana:
+* [ProblemChild Release Blog](link to blog)
+
+
+*Note: in order to use these ML features, you must have a platinum or higher [subscription](https://www.elastic.co/subscriptions)*
+*Note: the ML features are considered experimental in Kibana as well as this rules CLI*
+
+
+## Detailed steps
+
+#### 1. Upload and setup the model file and dependencies
+
+Run `python -m detection_rules es <args_or_config> experimental ml setup -t <release-tag>`
+
+*If updating a new model, you should first uninstall any existing models using `remove-model`*
+
+You can also upload files locally using the `-d` option, so long as the naming convention of the files match the 
+expected pattern for the filenames.
+
+#### 2. Update index pipeline configuration
+
+TODO
+
+#### 3. Refresh your indexes
+
+You can optionally choose to refresh your packetbeat index mapping from within Kibana:
+* Navigate to `Stack Management > (Kibana) Index Patterns` 
+* Select the appropriate indexes
+* Click `refresh field list`
+
+#### 4. Verify enrichment fields
+
+TODO
```

Comment:
@ajosh0504 can you add a snippet here on fields to look for to verify it is working